src/script/components/Modals/UserModal/UserModal.test.tsx

```diff
@@ -19,43 +19,23 @@
 
 import {QualifiedId} from '@wireapp/api-client/src/user';
 
-import TestPage from 'Util/test/TestPage';
 import {UserState} from 'src/script/user/UserState';
 import {TeamState} from 'src/script/team/TeamState';
 import {Core} from 'src/script/service/CoreSingleton';
-import UserModal, {UserModalProps} from './UserModal';
+import UserModal from './UserModal';
 import {UserRepository} from 'src/script/user/UserRepository';
 import {ActionsViewModel} from 'src/script/view_model/ActionsViewModel';
 import {User} from 'src/script/entity/User';
-
-class UserModalPage extends TestPage<UserModalProps> {
-  constructor(props?: UserModalProps) {
-    super(UserModal, props);
-  }
-
-  getWrapperElement = () => this.get('div.user-modal');
-  getStatusModalText = () => this.findByTestId('status-modal-text');
-}
+import {render, waitFor} from '@testing-library/react';
 
 describe('UserModal', () => {
-  it('does not render when user id is not passed', async () => {
```

Comment:
This seems not to be a case anymore since we're always passing qualified id object as `userId` prop. Typescript does not allow to pass `null` there.